packages/mask/src/components/InjectedComponents/ProfileCard/ProfileCardTitle.tsx

```diff
@@ -74,23 +75,27 @@ export const ProfileCardTitle: FC<Props> = ({
     }, [socialAddressList])
 
     return (
-        <div className={cx(classes.title, className)} {...rest}>
-            <PluginWeb3ContextProvider pluginID={NetworkPluginID.PLUGIN_EVM} value={{ chainId: ChainId.Mainnet }}>
+        <PluginWeb3ContextProvider pluginID={NetworkPluginID.PLUGIN_EVM} value={{ chainId: ChainId.Mainnet }}>
```

Comment:
Why not move this context provider outside as the top node of `ProfileCard`. Now it just covers a small area of the plugin.

Reply:
done